### ISSUE TITLE
IndicatorSeekbar breaks when there are only two tickmarks

### ISSUE THREAD

**Issue by Quelake** [warkiz/IndicatorSeekBar#132]
I am using the IndicatorSeekbar for a variable number of options. The smallest number of options is two. 'Snapping' to either of the available options is then no longer working.

For example:
````
<com.warkiz.widget.IndicatorSeekBar
                android:id="@+id/tempseekbar"
                android:layout_width="match_parent"
                android:layout_height="wrap_content"
                android:layout_marginTop="16dp"
                android:layout_below="@id/action_button"
                app:isb_max="100"
                app:isb_min="0"
                app:isb_progress="0"
                app:isb_seek_smoothly="true"
                app:isb_ticks_count="2"/>
````

**Comment by Quelake** [comment]
I have fixed it in my own repo and created a pull request to add this functionality, please take a look. The update is small, but really important for me to have in here. Thank you!

**Comment by warkiz** [comment]
@Quelake I have merge your PR and will release a new version in this week, really thanks for your RP and issue, it saves me lots of time.

**Comment by warkiz** [comment]
@Quelake Hi , I have reverted this PR #133 because something wring happened when use in pre-2.1.1version, now I update this to v2.1.2 to fixed this, You can use  v2.1.1 which has merge your PR.
Thanks again.

**Comment by YodaEmbedding** [comment]
This seems to still occur in v2.1.2.
